src/rgw/rgw_op.cc

```diff
@@ -7525,7 +7527,7 @@ int RGWHandler::init(rgw::sal::Store* _store,
 
 int RGWHandler::do_init_permissions(const DoutPrefixProvider *dpp, optional_yield y)
 {
-  int ret = rgw_build_bucket_policies(dpp, store, s, y);
+  int ret = rgw_build_bucket_policies(s, store, s, y);
```

Comment:
Should this one be dpp and not s?

Reply:
Yes, I'll make a note to change this when I go ahead and squash the commits.